api/nakadi-event-bus-api.yaml

```diff
@@ -474,7 +501,7 @@ definitions:
         type: string
       offset:
         type: string
-  SimpleStreamEvent:
+  EventStreamAtom:
```

Comment:
##### @tmuehl @CyberDem0n 

Please vote on the name:
* `EventStreamAtom` - as this is an atomic part of the stream (one cannot send less then this)
* `EventStreamBatch` - maybe not so pathetic as `atom` :D